DZ_8/controller.py

```diff
@@ -0,0 +1,95 @@
+import view
+import model
+import bd 
+import sqlite3
+from sqlite3 import Error
+import logger as log
+
+FILENAME_DATA = 'phonebook.txt'
+
+
+def select_action():
+    while True:
+        data = view.show_main_menu()
+        if not data or data == 0: break
```

Comment:
Согласно PEP8, так делать не принято.
```suggestion
        if not data or data == 0:
            break
```